.github/workflows/tests_linux.yml

```
@@ -343,16 +343,38 @@ jobs:
           pip install --upgrade tensorflow~=$TF_VERSION keras~=$TF_VERSION
 
       - name: Install backend device
+        if: ${{ matrix.pl_backend != "all" }}
```

CI doesn't work because of these. I think single quotes are required.
```suggestion
        if: ${{ matrix.pl_backend != 'all' }}
```

---

Thank you for that.